### ISSUE TITLE
Add a black UI theme

### ISSUE THREAD

**Issue by akmLaVx** [AdguardTeam/ContentBlocker#211]
### Problem Description

On Samsung phones or other devices w/OLED screens, switching to pitch black allows the device to save on battery charge.

### Proposed Solution

Add a pitch-black color theme to AdGuard Content Blocker (and probably AdGuard for Android).

### Additional information
Requested by user 1638590

**Comment by ameshkov** [comment]
> and probably AdGuard for Android

It's already there

**Comment by fghrthjt** [comment]
Even in AdGuard for Android

a dark subject, not black.

**Comment by rentalweb** [comment]
A dark theme would be VERY helpful!

**Comment by LuisAPI** [comment]
Agreed, dark theme in AdGuard would complement Samsung Internet's heavily advertised built-in dark theme feature.
